tests/exec-steps.sh

```diff
@@ -25,6 +25,10 @@ for STEP in /var/lib/tests/step-*.sh; do
     rm -rf ~/.githooks
     rm -rf /tmp/*
 
+    # Fake a default installation directory (for tests which do not install)
+    mkdir -p ~/.githooks
+    git config --global githooks.installDir ~/.githooks
```

Comment:
Please don't set this here, as this would make all tests **not** test the case when we don't have this set up.